app/scripts/constants/contracts.ts

```diff
@@ -4,7 +4,7 @@ export const SINGLE_CALL_BALANCES_ADDRESSES = {
   [CHAIN_IDS.MAINNET]: '0xb1f8e55c7f64d203c1400b9d8555d050f94adf39',
   [CHAIN_IDS.GOERLI]: '0x9788C4E93f9002a7ad8e72633b11E8d1ecd51f9b',
   // TODO(SEPOLIA) There is currently no balance call address for Sepolia
-  [CHAIN_IDS.SEPOLIA]: '',
+  // [CHAIN_IDS.SEPOLIA]: '',
```

Comment:
Curious question: Why are we keeping it commented rather than removing it?

Reply:
I kept it commented because there’s a TODO above it.